macros/trace_object/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "mmtk-macro-trace-object"
```

Comment:
When the next version is published, this crate will be visible on https://crates.io/crates/mmtk-macro-trace-object

We'd better not litter crates.io with too many crates.

Reply:
Does this macro crate get published automatically when mmtk-core is published? Do we need to publish this macro crate separately?

Reply:
I tried `cargo publish --dry-run` in `mmtk-core`.  It shows that I must push `mmtk-macro-trace-object` to crates-io before pushing `mmtk-core`, because `mmtk-core` depends on `mmtk-macro-trace-object`.

However, it is okay to `cargo publish --dry-run` in `mmtk-core/macros/trace_object`, because it has no unresolved dependencies.

There is an open issue about this topic: https://github.com/rust-lang/cargo/issues/1169

Reply:
I see. I probably need to update our github action for publishing.

Reply:
Done.